### PR TITLE
Update de.json

### PR DESCRIPTION
**Pull request by eth3rgeist**
Refined some german phrases.

Signed-off-by: Knuth <knuth@ethergeist.de>

### CODE REVIEW

**Comment by ttheuer** [comment]
Typo: `Aktuallisieren` should only have on `l` (lower case L) according to the Duden.

**Comment by SISheogorath** [comment]
Just as hint: We usually use POEditor as our translation platform. See: https://translate.codimd.org

It's the preferred way to add/change translations. Anyway, I'm quite sure we can make it to merge this PR as soon as some people checked it.

**Comment by eth3rgeist** [comment]
Fixed the typo.

File: locales/de.json [hunk 1]
```diff
@@ -43,17 +43,17 @@
     "Export": "Exportieren",
     "Import": "Importieren",
     "Clipboard": "Zwischenablage",
-    "Download": "Download",
+    "Download": "Herrunterladen",
```
Comment (SISheogorath) [inline locales/de.json]: Should be `Herunterladen`

**Comment by SISheogorath** [comment]
Since the changes are not coming in, please feel free to contribute translations on our translation platform (POEditor): https://translate.codimd.org

**Pull request closed without merging**